src/modules/twap/hooks/useTwapOrdersExecutions.ts

```diff
@@ -55,10 +56,14 @@ export function useTwapOrdersExecutions(ids: string[]): TwapOrdersExecutionMap {
         const executedBuyAmount = sumChildrenAmount(children, 'executedBuyAmount').toString()
         const executedSellAmount = sumChildrenAmount(children, 'executedSellAmount').toString()
         const executedFeeAmount = sumChildrenAmount(children, 'executedFeeAmount').toString()
+        const confirmedPartsCount = children.filter((order) => CONFIRMED_STATES.includes(order.status)).length
```

Comment:
A lot of this logic might bee generic, not just TWAP. No need to do a big refactor since we only have TWAP, but parent/child relationship is for more kinds of orders